src/hidimstat/base_variable_importance.py

```diff
@@ -131,3 +132,71 @@ def _check_importance(self):
             raise ValueError(
                 "The importances need to be called before calling this method"
             )
+
+    def plot_importance(
+        self,
+        feature_names=None,
+        ax=None,
+        ascending=False,
+        **kwargs,
+    ):
+        """
+        Plot feature importances as a horizontal bar plot.
+
+        Parameters
+        ----------
+        importances : array-like of shape (n_features,) or (n_repeats, n_features)
+            Feature importance scores. If 2D, each column represents a different repetition.
+        feature_names : list of str
+            Names of the features.
+        ax : matplotlib.axes.Axes or None, default=None
+            Axes object to draw the plot onto, otherwise uses the current Axes.
+        ascending: bool, default=False
+            Whether to sort features by ascending importance.
+        **kwargs : additional keyword arguments
+            Additional arguments passed to seaborn.barplot.
+
+        Returns
+        -------
+        ax : matplotlib.axes.Axes
+            The Axes object with the plot.
+        """
+        try:
+            import matplotlib.pyplot as plt
+            import seaborn as sns
+        except ImportError:
+            raise Exception("You need to install seaborn for using this functionality")
+
+        self._check_importance()
+        if ax is None:
+            _, ax = plt.subplots()
+        if feature_names is None:
+            feature_names = [f"x_{i}" for i in range(self.importances_.shape[-1])]
+
+        if self.importances_.ndim == 2:
+            df_plot = {
+                "Feature": feature_names * self.importances_.shape[0],
+                "Importance": self.importances_.flatten(),
+            }
+        else:
+            df_plot = {
+                "Feature": feature_names,
+                "Importance": self.importances_,
+            }
+
+        df_plot = pd.DataFrame(df_plot)
+
+        # Sort features by decreasing mean importance
```

Comment:
Normally, the importance is only one value in the API.
The other case, if the model uses CV or bootstrapping, but in this case, the importance won't have multiple values, there will be another attribute for it.

Reply:
Discussed in #449